.github/workflows/build.yml

```diff
@@ -25,10 +25,19 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
+      with:
+        fetch-depth: 2
     - name: 'Install Dependencies'
-      run: sudo apt-get update && make ciprepare
+      run: |
+        sudo apt-get update && make ciprepare
+        sudo apt-get install libssl-dev
+        cargo install cargo-tarpaulin
```

Comment:
this should be restored since the install happens via the make ciprepare target

Reply:
please restore this and do the cargo install in the Makefile - otherwise this causes the tests to run longer